src/shell/shell_cmds.cpp

```diff
@@ -790,6 +834,7 @@ void DOS_Shell::CMD_LS(char *args)
 
 	const int column_sep = 2; // chars separating columns
 	const auto cell_widths = map_name_lengths(results, column_sep);
+	const auto column_widths = calc_column_widths(cell_widths, column_sep + 1);
```

Comment:
Yeah, here's where my comments on the `map_name_lengths` function really hit home.. because `auto` completely hides what we're getting back.. I would assume it's a std::map.